### ISSUE TITLE
"Manage facilities" is not properly selected when editing facility information

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4096]
## Area or Component
"Manage facilities" section of the settings - facility form

## Quick Summary
When navigating to the facility form, the "manage facilities" sub-navigation is not selected. The element's state is not properly conveyed to accessible users (or to sighted users) making the pages more difficult to navigate.

### How to Replicate
1. Enter settings
2. Choose "Manage facilities"
3. Select one of the facilities
4. Notice "Manage facilities" in the mini nav bar is no longer selected

## Accessibility Violation
States/Properties: The element has missing or incorrect states or properties: https://cdc-devtools.dequecloud.com/rule-help/en/aria-state-property-missing

## Action Requested
Ensure that "manage facilities" remains selected when choosing a sub-facility.

## Screenshots (optional)
<img width="1000" alt="Screen Shot 2022-07-27 at 4 16 15 PM" src="https://user-images.githubusercontent.com/80282552/181388502-a5ca12e3-dca8-4fac-850e-449c482ecb3c.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1346058064&range=7:7
